tests/internal.spec.js

```diff
@@ -22,19 +22,19 @@ describe('Can handle', function () {
   it('missing username', function () {
     expect(function () {
       nclWrapper();
-    }).to.throw('AssertionError');
+    }).to.throw();
   });
 
   it('missing password', function () {
     expect(function () {
       nclWrapper(testData.username);
-    }).to.throw('AssertionError');
+    }).to.throw();
   });
 
   it('missing email', function () {
     expect(function () {
       nclWrapper(testData.username, testData.password);
-    }).to.throw('AssertionError');
+    }).to.throw();
```

Comment:
should we use the error message here:`
.to.throw('must include email in auth')`

Reply:
@jEnbuska Yes, my bad I removed those because Node 4 was still throwing `AssertionError`. Added error messages back 👍